genologics/entities.py

```diff
@@ -765,24 +755,30 @@ class Entity(object):
 
     _TAG = None
     _URI = None
+    _PREFIX=None
 
-    def __new__(cls, lims, uri=None, id=None):
+    def __new__(cls, lims, uri=None, id=None, create_new=None):
```

Comment:
this create_new seems to be used mostly as a boolean. If that is the case, would it be possible to initialize it to false, and treat it as a boolean ?